eth/tracers/native/bundler_collector.go

```diff
@@ -128,7 +128,7 @@ func (b *bundlerCollector) isEXTorCALL(opcode string) bool {
 // [OP-062]
 func (b *bundlerCollector) isAllowedPrecompile(addr common.Address) bool {
 	addrInt := addr.Big()
-	return addrInt.Cmp(big.NewInt(0)) == 1 && addrInt.Cmp(big.NewInt(10)) == -1
+	return (addrInt.Cmp(big.NewInt(0)) == 1 && addrInt.Cmp(big.NewInt(10)) == -1) || addrInt.Cmp(big.NewInt(256)) == 0
```

Comment:
It might be more straightforward to have `0x100` than `256`

Reply:
Agreed. Originally, it was 256 for consistency, but it was changed to 0x0100 for clarity.